src/AppBundle/Tests/Controller/InterviewControllerTest.php

```diff
@@ -397,6 +431,60 @@ public function testSchedule()
         \TestDataManager::restoreDatabase();
     }
 
+    public function testWantTeamInterest()
+    {
+        $ACT = new AdmissionControllerTest();
```

Comment:
Her er det kjempebra at du/dere tenker gjenbruk av kode! `countRows` er noe som blir gjort mange steder, så det gir mening å gjenbruke denne koden.

Men hører egentlig denne (generelle) funksjonaliteten hjemme i `AdmissionControllerTest`?

Nå opprettes det en avhengighet mellom to testklasser, `AdmissionControllerTest` og `InterviewControllerTest`, som ikke er helt ønskelig. I steden foreslår jeg heller at vi utvider testrammeverket med funksjoner som `countRows` slik at disse kan gjenbrukes i alle testklassene (uten å skape avhengigeter mellom klassene).

Jeg har begynt på en slik utvidelse i PR #496 som kan tas i bruk her. Hvis du ikke vil vente på at #496 blir merget kan du bare kopiere `countRows`-funksjonen fra `AdmissionControllerTest` til denne klassen så lenge (for å kvitte oss med avhengigheten mellom testklassene), og så rydde opp senere etter #496 er merget.

Høres dette fornuftig ut?